tools/make/common.mk

```diff
@@ -113,6 +113,9 @@ Options:
 endef
 export USAGE_OPTIONS
 
+.PHONY: generate
+generate: kube-generate go.generate
```

Comment:
can you add a helper string that will show up with `make help` - `## Generate go code from templates and tags`

Reply:
done